### DISCUSSION TITLE
Should `winget show` use literal YAML field names instead of localizing them?

### DISCUSSION THREAD

**Discussion by florelis** [microsoft/winget-cli#1503] [Q&A]
# Brief description of your issue

When doing `winget show <pkg>` it uses localizable labels for each field, but maybe it should use the exact manifest field names (always English and no spaces)? Not sure if it's better to have it localized or match the manifests exactly.

# Steps to reproduce

`winget show vscode`

# Expected behavior

Maybe use the exact manifest field name, e.g.
```
LicenseUrl: https://code.visualstudio.com/license
```

# Actual behavior

Depending on language, something like:
```
License Url: https://code.visualstudio.com/license
or
Dirección URL de la licencia: https://code.visualstudio.com/license
```
(omitting the remaining fields for brevity)

# Environment

```none
[winget --info]
Windows Package Manager version
Windows: Windows.Desktop version
Package: Microsoft.DesktopAppInstaller version

Any other software?
```

**Comment by vedantmgoyal9** [comment]
I would vote for "literal manifest field names".
